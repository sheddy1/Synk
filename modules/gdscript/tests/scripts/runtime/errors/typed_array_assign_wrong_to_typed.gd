func test():
	var typed: Array[GlobalClass1] = [GlobalClass2.new() as GlobalBase]
	print('not ok')
