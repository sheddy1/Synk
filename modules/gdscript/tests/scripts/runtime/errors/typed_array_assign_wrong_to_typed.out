GDTEST_RUNTIME_ERROR
>> ERROR
>> Method/function failed.
>> Unable to convert array index 0 from "RefCounted (GlobalClass2)" to "RefCounted (GlobalClass1)".
not ok
